alerts/cluster-network-operator/SouthboundStaleAlert

```diff
@@ -0,0 +1,23 @@
+# Southbound Stale Alert 
+
+## Meaning 
+
+The alert `SouthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
```

Comment:
I think this is better? WDYT? I dont like the word synching. Its really translating higher level objects in nb db to lower level objects and not technically synching.

The alert `SouthboundStale` is triggered when OVN Southbound SB has not been written to for 2 minutes or longer by OVN northd. Therefore, any networking control plane changes are not being updated in Southbound DB for the clusters nodes to consume.

Reply:
Changing to this now